sdk/framework/openapi.go

```diff
@@ -883,6 +955,10 @@ func cleanResponse(resp *logical.Response) *cleanedResponse {
 //	postSysToolsRandomUrlbytes_2
 //
 // An optional user-provided suffix ("context") may also be appended.
+//
+// Deprecated: operationID's are now populated using constructOperationID();
+// This function is here for backwards compatibility with older plugins and
+// will soon be reomoved.
```

Comment:
Typo: 
```suggestion
// will soon be removed.
```

but also, can this really be removed any time soon?

Should it not really remain until all plugins built using older `sdk` versions are wholly unsupported, for which, AFAIK, there is no written policy - so not soon, at least?

Also, this function currently has the role of fixing up duplicate operation IDs by appending a numeric suffix, and that is now being bypassed when pre-calculated OperationIDs are used. Ideally, that functionality would be retained - because, even though it generally should not happen, humans occasionally make mistakes, and it's a lot easier to find them and correct them if the duplicate operation IDs are being visibly given numeric suffixes, rather than silently overwriting each other's request/response schemas.

Reply:
Fair point, I suppose it will just have to stay here for a while, but will be marked deprecated, I'll update the comment.

Reply:
> Also, this function currently has the role of fixing up duplicate operation IDs by appending a numeric suffix, and that is now being bypassed when pre-calculated OperationIDs are used. Ideally, that functionality would be retained - because, even though it generally should not happen, humans occasionally make mistakes, and it's a lot easier to find them and correct them if the duplicate operation IDs are being visibly given numeric suffixes, rather than silently overwriting each other's request/response schemas.

Currently, my plan is to address all such potential ambiguities in related PR's that adjust `OperationPrefix  / OperationSuffix / Action` for various built-in plugins. OpenAPI does not permit duplicate `operationId` values, so in case of duplicates (at least within the built-in plugins), the spec generated using [gen_openapi.sh](https://github.com/hashicorp/vault/blob/main/scripts/gen_openapi.sh) will be invalid and will be flagged by one of the CI jobs that we are planning to add in the future.